### PR TITLE
Pom gitignore update

### LINKED ISSUE #4: Set Up Project
create repo
connect to intelliJ
connect to azure
set up CI

### PR DESCRIPTION
**Pull request by maishabd23**
Used the following link to update pom files for root and module directories: https://spring.io/guides/gs/multi-module/

### CODE REVIEW

File: project9/pom.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  <parent>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-parent</artifactId>
+      <version>3.1.0</version>
+      <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>com.example.project9</groupId>
+  <artifactId>project9</artifactId>
+  <version>1.0</version>
+
```
Comment (thanujasiva) [inline project9/pom.xml]: add this as well: `<name>project9</name>`

**Comment by shrimeichock** [comment]
change  <groupId>com.example.project9</groupId> to new package name

**Review by waheebh1 — COMMENTED** [review]
i think you need to keep the pom.xml at the root with the contents of the pom.xml that is in the project9 folder, then remove the one in project9 folder afterwards

**Comment by thanujasiva** [comment]
> i think you need to keep the pom.xml at the root with the contents of the pom.xml that is in the project9 folder, then remove the one in project9 folder afterwards


It looks like we need both pom.xml files
* the root file pom.xml for CI
* the project9 pom.xml to be able to use the dependencies when working on the code

**Review by thanujasiva — APPROVED** [review]

**Comment by waheebh1** [comment]
> > i think you need to keep the pom.xml at the root with the contents of the pom.xml that is in the project9 folder, then remove the one in project9 folder afterwards
> 
> It looks like we need both pom.xml files
> 
> * the root file pom.xml for CI
> * the project9 pom.xml to be able to use the dependencies when working on the code

im not sure but i think it should work with the root file pom.xml since that's how i did it in the lab

**Review by thanujasiva — APPROVED** [review]

**Review by shrimeichock — APPROVED** [review]
good

**Review by waheebh1 — APPROVED** [review]

**Comment by thanujasiva** [comment]
Relates to Issue https://github.com/waheebh1/online-bookstore/issues/4

**Pull request merged**